### DISCUSSION TITLE
Used space in OSS is 2x of JuiceFS

### DISCUSSION THREAD

**Discussion by wyujie** [juicedata/juicefs#585] [General]
**What happened**:
阿里云储存使用空间大于本地4倍以上

**What you expected to happen**:
空间占用一样多

**Environment**:
- CentOS 7
- Juicefs 0.14.2

**Comment by davies** [comment]
@wyujie Have you tried collect potential garbage using `juicefs gc` (maybe `--compact`)?

**Reply by wyujie** [reply]
Hi davies, thanks for your kind reply. 

I have tried  `juicefs gc --compact`. Then I use `du` to stat the mount floder. The output number is expected. But the number of used capacity of OSS showing in the aliyun admin site are still the large number, 2x of local used space. 

I think the leaked chunk are still existed in aliyun OSS after running `juicefs gc --compact`?  What do you think?


Regards
wyujie

**Reply by davies** [reply]
What's the output of `juicefs gc --compact` ? you need to add `--delete` to delete the found garbage.

**Reply by wyujie** [reply]
**What's the output of juicefs gc --compact ?**
Get output like below:
```
2021/07/11 04:20:44.810902 juicefs[13596] <INFO>: found leaked chunk c31364_1
2021/07/11 04:20:45.002692 juicefs[13596] <INFO>: found leaked chunk c31995_1
2021/07/11 04:20:45.186046 juicefs[13596] <INFO>: found leaked chunk c32444_1
2021/07/11 04:20:45.361131 juicefs[13596] <INFO>: found leaked chunk c31675_1
2021/07/11 04:20:45.469491 juicefs[13596] <INFO>: found leaked chunk c32228_1
2021/07/11 04:20:45.694713 juicefs[13596] <INFO>: found leaked chunk c31463_1
2021/07/11 04:20:45.836390 juicefs[13596] <INFO>: found leaked chunk c31360_1
2021/07/11 04:20:45.944757 juicefs[13596] <INFO>: found leaked chunk c31820_1
```

**you need to add --delete to delete the found garbage**
It looks work. Thank you very much !

**Comment by copecog** [comment]
I'm having a similar problem as this. Juicefs version 1.1.2+2024-02-04.8dbd89a on minio and perconadb xtradb cluster. I can't run `juicefs gc` because my minio is working to answer the listallobjects query but juicefs gc times out after 30 seconds. Is there a way to raise that?
